CMakeLists.txt

```diff
@@ -147,6 +147,13 @@ if(DESKTOP AND APPLE)
   endif()
 endif()
 
+if(APPLE)
+  # For iOS and macOS, disable nullability completeness warning, as it makes the
+  # build output for our Objective-C++ files much too verbose.
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-nullability-completeness")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-nullability-completeness=")
```

Comment:
Why does one have an equal sign, and the other doesn't?

Reply:
Because of a typo. :)